src/Core/src/Handlers/View/ViewHandler.Windows.cs

```diff
@@ -1,31 +1,39 @@
 ﻿#nullable enable
 using System;
 using System.Collections.Generic;
-using System.ComponentModel;
 using Microsoft.UI.Xaml;
-using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Input;
 using PlatformView = Microsoft.UI.Xaml.FrameworkElement;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class ViewHandler
 	{
+		static Dictionary<PlatformView, ViewHandler>? FocusManagerMapping;
```

Comment:
Should this be a `ConditionalWeakTable`? If `DisconnectingHandler()` were never called (which can be the case), would this persist both the `PlatformView` and `ViewHandler` in memory indefinitely?